test/unit/models/flag.test.js

```diff
@@ -0,0 +1,46 @@
+const chai = require("chai");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+
+const addUser = require("../../utils/addUser");
+const cleanDb = require("../../utils/cleanDb");
+
+const flagModel = require("../../../models/flag");
+const firestore = require("../../../utils/firestore");
+const flagFirestore = firestore.collection("featureFlags");
+
+const flagData = {
```

Comment:
Need to remove this and use the fixture.

Reply:
Tried that but whenever importing fixture, it is giving data as undefined so not able to use that.

Reply:
We are exporting a function right? So call it will be something like this. `require("../../path")();`